source/ConversationPanel.cpp

```diff
@@ -202,8 +202,9 @@ void ConversationPanel::Draw()
 bool ConversationPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command, bool isNewPress)
 {
 	// Map popup happens when you press the map key, unless the name text entry
-	// fields are currently active.
-	if(command.Has(Command::MAP) && !choices.empty())
+	// fields are currently active. The name text entry fields are active if
+	// choices is empty and we aren't at the end of the conversation.
+	if(command.Has(Command::MAP) && (!choices.empty() || node < 0))
```

Comment:
Is it possible to map the Command::MAP to SDLK_RETURN or SDLK_KP_ENTER or to the 'd' key? (and then have the boardingPannel from the Exit-function called on line 214) to open on top of the map?

Reply:
Not quite sure what you mean by that. Are you saying that if the map is opened at the end of the conversation, it should also exit out of the conversation entirely?

Reply:
What I meant is what happens if the player changes his/her preferences to use one of the keys from line 212 as a key to open the map. I was wondering if that could lead to a situation where the MapDetailPanel() gets pushed here on line 208 and the Exit() function on line 214 also doing something after that like pushing a BoardingPanel())

Reply:
Not sure, but if there are issues with that then there were issues before as well.

Reply:
Agreed, that would not be a new issue, so we can address that separately.